documentation/src/main/markdown/packagemgrs/opam.md

```diff
@@ -0,0 +1,37 @@
+# Opam Support
+
+## Related properties
+
+[Detector properties](../properties/detectors/opam.md)
+
+[detect_product_short] has two detectors for Opam:
+
+* OPAM CLI Detector
+* OPAM Lock Detector
+
+## OPAM CLI Detector
+
+* This detectable executes opam commands to discover dependencies of opam projects.
+
+* This OPAM detector will be executed on your project if [detect_product_short] finds `.opam` file in your top level directory. It requires `opam`
+exe to be present on your $PATH. You can also override the location for `opam` exe by the OPAM path property.
```

Comment:
Minor tweak:

* The OPAM detector will be executed on your project if [detect_product_short] finds a `.opam` file in your top level directory. It requires `opam` exe to be present on your $PATH. You can also override the location for `opam` exe via the OPAM path property.